dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/dataset/metadata/inputRenderer/SuggestionInputFieldRendererTest.java

```diff
@@ -0,0 +1,91 @@
+package edu.harvard.iq.dataverse.dataset.metadata.inputRenderer;
+
+import com.google.common.collect.Lists;
+import edu.harvard.iq.dataverse.dataset.metadata.inputRenderer.suggestion.GrantSuggestionHandler;
+import edu.harvard.iq.dataverse.persistence.MocksFactory;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetField;
+import edu.harvard.iq.dataverse.persistence.dataset.FieldType;
+import edu.harvard.iq.dataverse.persistence.dataset.MetadataBlock;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class SuggestionInputFieldRendererTest {
+
+    private SuggestionInputFieldRenderer suggestionInputFieldRenderer;
+    private GrantSuggestionHandler suggestionHandler;
+
+    @BeforeEach
+    public void setup() {
+        suggestionHandler = Mockito.mock(GrantSuggestionHandler.class);
+    }
+
+    // -------------------- TESTS --------------------
+
+    @Test
+    public void createSuggestions_WithoutFilters() {
```

Comment:
I don't  see what is being tested here. We are mocking method return value and then checking what was returned?

Reply:
The only things we are testing here are the things like: Are we calling the desired methods, is there any exception along the path, is the method flow correct with provided arguments. You can't test much in this method without filters.